elastic/kibana
# [SLO][Embeddable] Fix Kibana reporting screenshot issue

Resolves https://github.com/elastic/kibana/issues/169716

## 🍒 Summary
The reporting plugin waits for a renderComplete signal when the embeddable has finished loading any data and is correctly rendered so the headless Chromium can take a screenshot. This PR calls the RenderCompleteDispatcher on the SLO Overview embeddable when all the child components have finished loading.

https://elastic.slack.com/archives/CFFQ7RP9B/p1614267987040400

## How to test

- Add an SLO panel to the dashboard
- click on share -> PDF reports > Generate PDF on the top right navigation bar
- No Timeout error should appear on the generated PDF

<img width="1407" alt="Screenshot 2023-10-26 at 15 05 05" src="https://github.com/elastic/kibana/assets/2852703/45c2c868-edbe-426e-bc05-5794c2bfa91e">
